### PR TITLE
Profile enhancements (see 1912)

### PR DESCRIPTION
**Pull request by mariavkourouni**
[Demo](https://mf-geoadmin3.dev.bgdi.ch/kom_profile/)

### CODE REVIEW

**Comment by cedricmoullet** [comment]
@mariavkourouni could you please update the demo link ?

**Comment by mariavkourouni** [comment]
@cedricmoullet now it works.

File: src/components/profile/ProfileService.js [hunk 1]
```diff
@@ -133,6 +133,33 @@ goog.require('ga_urlutils_service');
...
+          return Math.max.apply(null, this);
+        };
+        Array.prototype.min = function() {
+          return Math.min.apply(null, this);
+        };
+
+        //Highest & lowest elevation points
+        this.elPoints = function(data) {
+          if (data.length != 0) {
+            var elArray = new Array();
```
Comment (oterral) [inline src/components/profile/ProfileService.js]: Usually we use  `elArray = [];` to define an array

File: src/components/profile/ProfileService.js [hunk 2]
```diff
@@ -361,12 +391,70 @@ goog.require('ga_urlutils_service');
...
+
+          //Number for the distance
+          group.append('text')
+              .attr('class', 'ga-profile-distance')
+              .attr('font-size', '0.9em')
+              .attr('x', 412)
+              .attr('y', elevLabelY)
+              .style('text-anchor', 'left')
+              .text(measureFilter(this.dist, 'distance',
+                    'm', 2, true));
```
Comment (oterral) [inline src/components/profile/ProfileService.js]: Use `['km', 'm'] ` for the distance.

File: src/components/profile/ProfileService.js [hunk 3]
```diff
@@ -331,9 +361,9 @@ goog.require('ga_urlutils_service');
 
           //Icon for elevation up
           group.append('text')
-              .attr('class', 'ga-profile-icon')
+              .attr('font-family', 'FontAwesome')
```
Comment (oterral) [inline src/components/profile/ProfileService.js]: Why did you remove the `class`attribute ?

Comment (oterral) [inline src/components/profile/ProfileService.js]: I guess it's because it's not a `ga-profile-icon` :)

Use a `ga-profile-text`class to avoid duplicate style definition

Comment (mariavkourouni) [inline src/components/profile/ProfileService.js]: Because I added `font-size: 0.9em;` at the class `ga-profile-icon`
and I use this class for all the profile icons except for the icons elev-up and elev-down,
which need to have `font-size: 1.5em;`

**Comment by gjn** [comment]
@davidoesch Feedback?

**Comment by davidoesch** [comment]
perfect,, nicely done @mariavkourouni
